### DISCUSSION TITLE
theming value not provided for xxx falling back to yyy

### DISCUSSION THREAD

**Discussion by Link631** [microsoft/fluentui#18508] [Q&A]
Hi,

when loading a theme, I get warning for each theme property I didnot provide. It is just for debug mode, but still, during development I still do not want to get those warnings, because the default values are just fine for me.

As a solution, I would expect to have a function for creating a blank theme with all default values. For pallettes and also for semantic.

How can I create a theme that has the default values set in the same way, as the fallbacks would set the values?

If there is not such a function, than it would make sense to at least reduce that warning into an info, which describes how to create a theme with default values set.

In addition I just created a theme via createTheme. I then even called mergeTheme. As a result i have a theme with all properties set, even Sematic Colors are set.

Why am i getting warnings like: value not privded for error? Which value in the theme should I set to make this warning disappear??

![image](https://user-images.githubusercontent.com/54800010/121479181-118d9400-c9ca-11eb-9010-2ab1c5a55012.png)

Thanks

**Comment by micahgodbolt** [comment]
Can you create a codepen of your current setup, with notes on what you're trying to have happen?
